guide/creating-your-bot/collectors.md

```diff
@@ -79,7 +79,7 @@ message.channel.send(item.question).then(() => {
 
 In this filter, you iterate through the answers to find what you want. You would like to ignore case because simple typos can happen, so you convert each answer to its lowercase form, and check if it's equal to the response in lowercase form as well. In the options section, you only want to allow one answer to pass through, hence the `maxMatches: 1` setting.
 
-The filter looks for messages that match one of the answers in our array of possible answers in order to pass through the collector. In the options, in the second parameter, it's specified that only a maximum of 1 message can go through the filter successfully, before the promise will successfully resolve. In the errors section, it's specified that time will cause it to error out, which will cause the promise to reject if 1 correct answer is not received within the time limit of 1 minute. As you can see, there is no `collect` event, so you are limited in that regard.
+The filter looks for messages that match one of the answers in our array of possible answers in order to pass through the collector. In second parameter of the options, it's specified that only a maximum of 1 message can go through the filter successfully before the promise will successfully resolve. In the errors section it's specified that time will cause it to error out, which will cause the promise to reject if 1 correct answer is not received within the time limit of 1 minute. As you can see, there is no `collect` event, so you are limited in that regard.
```

Comment:
"In the second parameter of the options" makes it sound like the options is a function that takes two parameters; my mistake for not realizing this when you asked. I think reverting it to the original wording would be better, but word it as "In the options (the second parameter), ..."